app/case/create_case.mustache

```diff
@@ -1,13 +1,15 @@
+
```

Comment:
We seem to be using 2 and 4 space tabs in different projects. We should probably set a standard for the project. Personally I prefer 2 space tabs for html but I think i've seen in most of the other projects that we have used 4 space tabs.